frontend/doc-site/04__Analyze Public Data/4_2__Gene Expression Documentation/4_2_3__Gene Expression Data Processing.mdx

```diff
@@ -75,58 +75,31 @@ Only cells from sequencing assays that measure gene expression and don't require
-
-<Image src={"/doc-site/4-2_figure6.png"} />
-<Image src={"/doc-site/4-2_figure7.png"} />
-
-**Figure 6.** Visual representation of rankit normalization.
-
-This method accounts for sequencing depth by scaling gene expression to the approximate range of 0 to 6. A high gene-cell value ( > 5 ) indicates that the gene is amongst genes that are the highest expressed in that cell, and similarly a low value ( < 1 ) indicates that the gene is amongst the lowest expressed genes in that cell.
+Read counts are normalized using a log transformation of pseudocounts per million (ln(CPM+1)).
+Normalized matrices from multiple datasets of the same tissue are then concatenated.
 
-Rankit normalization compresses highly expressed genes to the right tail of the standard normal distribution (see below), thus providing ideal values for using a color representation of gene expression and avoiding saturation that would otherwise be present.
-
-## Considerations
-
-Given that the normalization is a form of quantile normalization that uses a constant target distribution, there are a few important considerations while working with these values:
-
-- These values transform the biological/technical variance of gene expression to a constant variance from a normal distribution.
-- It has been proven that quantile normalization reduces technical variance and to a lesser extent biological variance (Abrams ZB et al.).
-- Individual gene expression values can produce spurious differential gene expression analysis (Zhao Y et al.).
-- This normalization is not designed to specifically account for batch effects, however the use of a constant target distribution mitigates batch effects when calculating average gene expression. This mitigation is stronger as the number of datasets per tissue increases.
+This normalization is not designed to correct batch effects; while normalization and subsequent aggregation of data across many observations (mean expression values) mitigates these effects to some degree, users should be aware that batch effects may still be present in the data.
 
 ## Removal of Noisy Ultra-low Expression Values
 
-After applying normalization, any gene/cell combination that had counts less or equal than 2 are set to missing data. This allows for removal of noise due to ultra-lowly expressed genes and provides a cleaner visualization.
+After applying normalization, any gene/cell combination counts less or equal than 2 are set to missing data. This allows for removal of noise due to ultra-lowly expressed genes and provides a cleaner visualization.
 
 ## Summarization of Data in Dot Plot
 
 For each gene/cell type combination the following values are represented in the dot plot, either visually or in text upon hover.
 
-- Gene expression (dot color) – the average rankit-normalized gene expression among genes that have non-zero values.
+- Gene expression (dot color) – the average ln(CPM+1)-normalized gene expression among genes that have non-zero values.
```

Comment:
it's not really CPM - it's sum-normalized to 10,000 counts prior to log transformation. don't really know what to call that

Reply:
Changed